homeassistant/components/device_tracker/__init__.py

```diff
@@ -57,6 +57,8 @@
 CONF_AWAY_HIDE = 'hide_if_away'
 DEFAULT_AWAY_HIDE = False
 
+EVENT_NEW_DEVICE = 'new_device_discovered'
```

Comment:
This event name is very ambiguous. Maybe rename it to `device_tracker_new_device` ?